src/components/layout/Navbar.tsx

```diff
@@ -73,10 +75,17 @@ const Navbar = () => {
           </PopoverContainer>
 
           <UserPointButton />
-          <LoginWithGoogleButton text='Login' />
+
+          {/* Show login button only if user is NOT authenticated and NOT on login page */}
+          {status === 'unauthenticated' && router.pathname !== '/login' && (
+            <LoginRedirectButton text='Login' />
+          )}
+
```

Comment:
Don't put any condition for login. handle all conditions inside login button only